ci/sitemap.js

```diff
@@ -0,0 +1,54 @@
+/**
+ * This script generates a sitemap.xml file for the website. It's meant to be run just before the site is deployed.
+ * 
+ * It reads all the .mdx files in the `pages` directory and generates a sitemap.xml file with the URLs.
+ */
+
+import { opendir, writeFile } from "fs/promises";
+import { join } from "path";
+
+async function walk(path, filter) {
+  let files = [];
+
+  const dir = await opendir(path);
+
+  for await (const item of dir) {
+    const file = join(dir.path, item.name);
+    if (item.isFile()) {
+      if (filter(file)) files.push(file);
+    }
+
+    if (item.isDirectory()) {
+      files = files.concat(await walk(file, filter));
+    }
+  }
+
+  return files;
+}
+
+const BASE_DOMAIN = "docs.discord.sex";
```

Comment:
shouldn't this be read from env

Reply:
the env only gives the domain that it's hosted on like `discord-userdoccers-775ddp0bn-discord-userdoccers.vercel.app`, doing this will make google explode i think